lib/iris/util.py

```diff
@@ -1599,3 +1599,10 @@ def demote_dim_coord_to_aux_coord(cube, name_or_coord):
     cube.remove_coord(dim_coord)
 
     cube.add_aux_coord(dim_coord, coord_dim)
+
+
+def is_dask_array(data):
+    result = False
+    if hasattr(data, 'compute'):
+        result = True
+    return result
```

Comment:
Actually, to be honest, if this turns into a one-liner, then why do we have this function call at all?

Just inline this directly into the code?